examples_tests/41.VisibilityBuffer/main.cpp

```diff
@@ -275,62 +272,76 @@ GPUMeshPacker packMeshBuffers(video::IVideoDriver* driver, core::vector<MbPipeli
 
     auto wholeMbRangeBegin = ranges.front().second;
     auto wholeMbRangeEnd = ranges.back().second;
+    const uint32_t meshBufferCnt = std::distance(wholeMbRangeBegin, wholeMbRangeEnd);
 
-    const uint32_t mdiCntTotal = mp.calcMDIStructCount(wholeMbRangeBegin,wholeMbRangeEnd);
+    const uint32_t mdiCntTotal = mp.calcMDIStructMaxCount(wholeMbRangeBegin,wholeMbRangeEnd);
+    //shouldn't it be `meshBufferCnt` instead of `mdiCntTotal`?
     auto allocData = core::make_refctd_dynamic_array<core::smart_refctd_dynamic_array<MeshPacker::ReservedAllocationMeshBuffers>>(mdiCntTotal);
```

Comment:
one alloc data points to multiple MDIs (range of), if so then yes you're probably right